opentasks/src/main/res/values-cy/strings.xml

```diff
@@ -0,0 +1,293 @@
+    <string name="classification_confidential">cyfrinachol</string>
+
+    <!-- Priority labels -->
+
+    <string name="priority_undefined">dim</string>
+    <string name="priority_low">isel</string>
+    <string name="priority_medium">arferol</string>
+    <string name="priority_high">uchel</string>
+    <string name="priority_urgent">brys</string>
+
+    <!-- Label for Settings -->
+
+    <string name="title_activity_settings">Gosodiadau</string>
+    <string name="action_settings">Gosodiadau</string>
+    <string name="action_add_local_list">Ychwanegu Rhestr</string>
+    <string name="visible_task_lists">Rhestrau wedi'u Dangos</string>
+    <string name="synced_task_lists">Rhestrau wedi'u Cysoni</string>
+
+    <!-- dialogs -->
+    <string name="task_list_settings_dialog_account_title">Dewisa gyfrif</string>
+
+    <!-- manage task list activity -->
+
+    <string name="activity_manage_list_btn_save">cadw</string>
+    <string name="activity_manage_list_btn_delete">dileu rhestr</string>
+    <string name="activity_add_list_title">Creu rhestr leol</string>
+    <string name="activity_edit_list_title">Gosodiadau rhestr</string>
+    <string name="task_list_name_caption">Enw</string>
+    <string name="task_list_color_caption">Lliw Rhestr</string>
+    <string name="task_list_no_sync">Noder: Mae hyn yn creu rhestr leol. Ni chaiff tasgau mewn rhestrau lleol eu cysoni.</string>
+
+    <!-- input dialog -->
+    <string name="task_list_name_dialog_hint">enw rhestr</string>
+    <string name="task_list_name_dialog_title">Rho enw restr</string>
+    <string name="task_list_name_dialog_error">Rho enw.</string>
+
+    <!-- delete dialog -->
+    <string name="task_list_delete_dialog_title">Dileu %s</string>
+    <string name="task_list_delete_dialog_message">Dileu'r rhestr a phob tasg ynddi?\nNoder: Nid oes modd adfer rhestr ar ôl iddi gael ei dileu.</string>
+    <string name="task_list_delete_toast">Dileuwyd %s</string>
+
+    <!-- Label for Expandable Group Descriptors -->
+    <string name="gd_by_due_date">Yn Ôl Dyddiad Terfyn</string>
+    <string name="gd_by_completed">Yn Ôl Cyflawnwyd</string>
+    <string name="gd_by_list">Yn Ôl Rhestr</string>
+
+    <!-- String for confirm delete dialog -->
+    <string name="confirm_delete_title">Cadarnhau dileu</string>
+    <string name="confirm_delete_message">Dileu'r dasg hon?</string>
+    <string name="confirm_delete_message_with_title">Dileu \"%1s\"?</string>
+    <string name="toast_task_deleted">Tynnwyd \"%1s\".</string>
+
+    <!-- Strings in the Task List Selection Fragment -->
+    <string name="task_list_selection_empty">Dim rhestr dasgau ar gael</string>
+    <string name="task_list_selection_title">Dewisa rhestrau i'w dangos</string>
+    <string name="task_list_selection_ok">IAWN</string>
+    <string name="task_list_selection_cancel">DIDDYMU</string>
+
+    <!-- Strings for Task List Widget -->
+    <string name="task_list_widget_title">Current tasks</string>
```

Comment:
Please remove untranslated strings. It's easier to spot them in Android Studio that way.

Reply:
Translated.